src/main/scala/viper/silver/plugin/standard/inline/InlineRewrite.scala

```diff
@@ -49,26 +53,28 @@ trait InlineRewrite extends PredicateExpansion {
     * If found, return the same statement with the expanded body of the predicate.
     *
     * @param stmts A Seqn whose statements will be traversed.
-    * @param method The method containing the inhale statement we wish to expand.
+    * @param method The member containing the inhale statement we wish to expand.
     * @param program The Viper program for which we perform this expansion.
     * @param cond The condition a predicate must satisfy to be expanded within an inhale statement.
     * @return The Seqn with all inhale, exhale, assert, and while loop statements expanded.
     */
-  private[this] def expandStatements(stmts: Seqn, method: Method, program: Program, cond: String => Boolean): Seqn =
+  private[this] def expandStatements(stmts: Seqn, member: Member, program: Program, cond: String => Boolean): Seqn = {
+    val noFoldUnfoldStmts = removeFoldUnfolds(stmts, cond)
```

Comment:
Just as whenever you expand expression you also want to get rid of `Unfolding`s, when you expand statements, you'd always want to get rid of `Fold` and `Unfold` as well.